### PR TITLE
MMB: add cities to the trip info panel

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: app/MMB/src/components/header/Location.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { Translation } from '@kiwicom/mobile-localization';
+import { Text, StyleSheet } from '@kiwicom/mobile-shared';
+
+function Location(props) {
```
Comment (tbergq) [inline app/MMB/src/components/header/Location.js]: why is there no flow type here?

File: app/MMB/src/components/header/Location.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <Text style={styleSheet.city}>
+      <Translation passThrough={props.data.airport.city.name} />
```
Comment (tbergq) [inline app/MMB/src/components/header/Location.js]: Shouldn't you use idx here?

Comment (mrtnzlml) [reply app/MMB/src/components/header/Location.js]: Why is Flow happy? 🤔

Comment (tbergq) [reply app/MMB/src/components/header/Location.js]: flow is broken 🙈

**Comment by tbergq** [comment]
Also, should there be a little more margin between the rows?

File: app/MMB/src/components/header/Location.js (hunk 2 above)
Comment (mrtnzlml) [reply app/MMB/src/components/header/Location.js]: I saw just a few cases where Flow was actually broken in last year. The problem is usually between the keyboard and chair like in this case - our `createFragmentContainer` HOC is the reason why it doesn' work... 🤦‍♂️

**Comment by mrtnzlml** [comment]
> Also, should there be a little more margin between the rows?

Will do. See my standup notes - separators are going to be next (it's a little bit more complicated).

**Comment by tbergq** [comment]
> separators are going to be next

that's fine

**Comment by mrtnzlml** [comment]
@tbergq `Location` types fixed. Thanks...

**Pull request merged**